src/client/Microsoft.Identity.Client/AppConfig/AssertionRequestOptions.cs

```diff
@@ -30,13 +31,19 @@ public class AssertionRequestOptions {
         /// <summary>
         /// Claims to be included in the client assertion
         /// </summary>
-        public string Claims { get; set; }
+        public string Claims { get; set; }        
 
         /// <summary>
         /// Capabilities that the client application has declared. 
         /// If the callback implementer calls the token issuer using another client application object 
         /// (e.g. ManagedIdentityApplication or ConfidentialClientApplication), the same capabilities should be used there.
         /// </summary>
         public IEnumerable<string> ClientCapabilities { get; set; }
+
+        /// <summary>
+        /// FMI path to be used for client assertion when exchanging MSI tokens.
```

Comment:
Is it only MSI token? could be a Oidc FIC token? 
It's up to the CredentialProvider?

Reply:
Cline generated this comment, let me update it.